src/firstvoices/backend/utils/CharacterSortOrderHelpers.py

```diff
@@ -0,0 +1,127 @@
+import re
```

Comment:
Let's stick to our file naming conventions and call this something like character_utils.py